test.py

```diff
@@ -118,3 +120,39 @@ def test_nested_input():
 def test_function_mixup():
     """Make sure that different functions with the same args are cached under different keys"""
     assert get_random_with_args(1) != get_another_random_with_args(1)
+
+
+def callTextChangeTest():
+    process = Popen(["python", "textChangeTest.py"], stdout=PIPE)
+    (output, err) = process.communicate()
+    exit_code = process.wait()
+    assert exit_code == 0
+
+    return output.strip()
+
+
+def changeText(orig, new):
```

Comment:
1. PEP8 for function name
2. The filename should be a module-level parameter, or a function argument, as you're referring to it more than once.
3. Try being clear about what the function does. Think about better parameter names or even a docstring. Perhaps `change_file_content_inplace(filename, pattern, repl)` that resonates with the `re.sub` documentation.